services/apps/portal/src/common/automapper/application-owner.automapper.profile.ts

```diff
@@ -36,6 +36,7 @@ export class ApplicationOwnerProfile extends AutomapperProfile {
                 uuid: a.corporateSummary.uuid,
                 fileName: a.corporateSummary.fileName,
                 type: DOCUMENT_TYPE.CORPORATE_SUMMARY,
+                description: null,
```

Comment:
is this null intentional?

Reply:
yes, corporateSummaries cannot have descriptions